### ISSUE TITLE
Interlinear type of reading Greek translation with dictionary

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#686]
I also saw saw a feature in a Bible app that I used on my iPhone some years ago, the name of the app I've forgotten. On the app, you can open a Greek translation side by side with other translations, and you can tap on any Greek word on that translocation and get it's meaning. This is different from the system available on your app whereby you tap on a strong number to get the meaning of a word. If you can incorporate these my suggestions into your app, aside from bettering it, it'll create a variety and widen the already distant advantage your app has over other bible apps.

**Comment by tuomas2** [comment]
Do we have hebrew/greek module that have strongs codes? If we have, perhaps this would be doable.

**Comment by bvahedy** [comment]
What's the difference whether you tap on the word or on its Strong's number, it gives you the same dictionary definition.
You can use HISB, it has both Greek and Strong's no. embedded.

**Comment by tuomas2** [comment]
I thin UX only diff. Related to #177

**Comment by bvahedy** [comment]
With BYZ interlinear you get all that information on the screen. Like this:
![Screenshot_20200530-092202_Bible Study](https://user-images.githubusercontent.com/46405809/83333757-1405cb80-a257-11ea-936c-6929f14d2f4b.jpg)

**Comment by machshev** [comment]
> What's the difference whether you tap on the word or on its Strong's number, it gives you the same dictionary definition.
> You can use BYZ , it has both Greek and Strong's no. embedded. 

For me the difference is in the normal modules. Being able to see the Strongs numbers themselves doesn't add any value to me. Only clutters the text and makes it more difficult to read. If the Strongs numbers are above or below the text, this is a little better... but still makes it difficult to speed read or skim the text when looking for a particular verse in a chapter, or trying to get an initial feel for a passage before honing in on detail.

Having the ability to read the text without Strongs numbers and still click on a word to see which original word is used gives the best of both worlds in my view.

**Comment by machshev** [comment]
> Via email:
> >  I also saw saw a feature in a Bible app that I used on my iPhone some years ago, the name of the app I've forgotten. On the app, you can open a Greek translation side by side with other translations, and you can tap on any Greek word on that translocation and get it's meaning. This is different from the system available on your app whereby you tap on a strong number to get the meaning of a word. If you can incorporate these my suggestions into your app, aside from bettering it, it'll create a variety and widen the already distant advantage your app has over other bible apps.
> 
> 

There is a similar app for Android called andbiblelexicon which might be worth looking at for inspiration. It's more focused to those learning and using the original languages directly. Unfortunately it's closed source. You can select several texts alongside each other and make your own interlinear version that scrolls together. On the desktop I use Bibletime which has similar features.

**Comment by tuomas2** [comment]
Related https://github.com/charlesLoder/greek-transliteration

**Comment by agrogers** [comment]
Doesn't AB enable that now... the word itself is a link and clicking the word shows the Greek or Hebrew. Strongs numbers do not need to be shown. And Ben's BYZ does it also. I feel like this can be closed.

**Comment by timbze** [comment]
@agrogers it may be what is now implemented. I was thinking of what StepBible has when I read OP... I can hover or click on the Greek word and the word in the other translations is highlighted.

![image](https://user-images.githubusercontent.com/35117769/190212517-4a44fa59-b6c0-43b4-a161-26a31d9b98c7.png)

**Comment by bvahedy** [comment]
I agree, HiSB does that very thing, after Toumas made HiSB available we could have closed this ticket as that module does exactly what the user requested.

**Comment by timbze** [comment]
@bvahedy HiSB is only Hebrew (not Greek) right?

**Comment by bvahedy** [comment]
@timbze , that function doesn't describe interlinear. Just highlighting words only tells the user that there is a dictionary definition behind that word.

**Comment by bvahedy** [comment]
HiSB has both embedded within it. See the image above.

**Comment by bvahedy** [comment]
For NT it even gives you morphology.

**Comment by uniqueid001** [comment]
For reference/comparison, [Blue Letter Bible](https://play.google.com/store/apps/details?id=org.blueletterbible.blb) has both forward and reverse interlinear, and a new "inline" interlinear display.  Their implementation is limited to KJV for the OT and NASB95 for the NT.
 
![Screenshot_20220915-075706](https://user-images.githubusercontent.com/12715614/191504805-f0e7ceb0-a37f-408f-b760-4d615715791e.png)
![Screenshot_20220915-075735](https://user-images.githubusercontent.com/12715614/191504823-9970c06c-b041-4bca-b84d-cc1ca9fb90ae.png)
